### DISCUSSION TITLE
V6 multiple submit with multiple schemas

### DISCUSSION THREAD

**Discussion by isshesure** [react-hook-form/react-hook-form#4664] [Q&A]
Hi, what's the recommended approach for multiple submit buttons with multiple schemas? Is it to `useState` with a current schema id, and then in each submit button `onClick` to call the setter to the appropriate schema id and then call `handleSubmit`. Then use the schema id as the `context` and in the `resolver` check the schema id?

**Comment by bluebill1049** [comment]
> Use the schema id as the context and in the resolver check the schema id?

prefer the above.

**Reply by isshesure** [reply]
ok, sorry it wasn't clear but that was one complete option - i'll try this `useState`/`context`/schema id approach, thanks
